doc/api/diagnostics_channel.md

```diff
@@ -1121,6 +1121,43 @@ While the diagnostics\_channel API is now considered stable, the built-in
 channels currently available are not. Each channel must be declared stable
 independently.
 
+#### Console
+
+`console.log`
+
+* `args` {any\[]}
+
+Emitted when `console.log()` is called. Receives and array of the arguments
```

Comment:
Maybe describe that this *is* the arguments array and therefore mutation is possible.